src/renderer/styles/img/illustrations/3.svg

```diff
@@ -0,0 +1 @@
 0.0504911,0.340491l0.34,1.06h22.87Z" transform="translate(24.0295, 59.4897)"></path><use fill="none" stroke="#191816" stroke-linejoin="round" transform="translate(24.0295, 59.4897)" xlink:href="#path-4"></use><path fill="#656769" stroke="#191816" stroke-linejoin="round" d="M0.00554032,0h24v3h-21.69l4.53591e-08,1.93822e-10c-1.28129,-0.00547511 -2.31554,-1.0486 -2.31007,-2.32989c1.43972e-05,-0.00336923 3.61339e-05,-0.00673843 6.521e-05,-0.0101076Z" transform="translate(29.5145, 77.49)"></path><path fill="#87898C" stroke="#191816" stroke-linejoin="round" d="M0,0h11.48v0.66v0c0,1.29235 -1.04765,2.34 -2.34,2.34c0,0 -7.10543e-15,0 -7.10543e-15,0h-9.14Z" transform="translate(53.52, 77.49)"></path><use fill="none" stroke="#191816" stroke-linejoin="round" transform="translate(24.0295, 59.4897)" xlink:href="#path-4"></use><path fill="#FF6242" stroke="#191816" stroke-linejoin="round" d="M1.57491,0.49c0.14,-0.49 0.74,-0.49 1.08,0c0.74,1 1.1,1.58 1.83,2.63l1.84,2.63c0.34,0.49 0.17,1.11 -0.31,1.11h-5.16l-5.40695e-08,-7.47697e-09c-0.503312,-0.0696002 -0.854906,-0.534038 -0.785305,-1.03735c0.00383998,-0.0277688 0.00894716,-0.0553475 0.0153055,-0.0826499l0.75,-2.57c0.3,-1.1 0.44,-1.62 0.74,-2.68Z" transform="translate(39.3551, 66.83)"></path><path fill="none" stroke="#FFFFFF" stroke-linecap="round" stroke-linejoin="round" d="M0,0l0.31,1.49" transform="translate(41.86, 69.21)"></path><path fill="#FFFFFF" d="M0.9,0.5c0.06,0.28 -0.08,0.5 -0.32,0.5l-4.02925e-08,-4.36994e-09c-0.266903,-0.0289472 -0.480612,-0.234436 -0.52,-0.5c-0.06,-0.28 0.08,-0.5 0.32,-0.5l3.49102e-08,4.23941e-09c0.26536,0.0322246 0.477398,0.236107 0.52,0.5Z" transform="translate(42, 71.7)"></path><path fill="#FFE500" stroke="#191816" stroke-linejoin="round" d="M6.74,3.77c0.41,2.07 -0.66,3.75 -2.42,3.75c-1.76,0 -3.5,-1.69 -3.91,-3.77c-0.41,-2.08 0.66,-3.75 2.41,-3.74c1.75,0.01 3.5,1.69 3.92,3.76Z" transform="translate(29.2, 63.59)"></path><path fill="#191816" d="M0.38,0c-0.24,0 -0.38,0.23 -0.33,0.5l6.893e-09,4.34007e-08c0.0427821,0.269371 0.259183,0.477606 0.53,0.51c0.23,0 0.37,-0.23 0.32,-0.5l1.33714e-08,7.91848e-08c-0.0446858,-0.264626 -0.254556,-0.47046 -0.52,-0.51Z" transform="translate(31.05, 65.87)"></path><path fill="#FFE500" stroke="#191816" stroke-linecap="round" stroke-linejoin="round" d="M0,3.5723e-08l2.5217e-08,6.16014e-08c0.198893,0.485866 0.648031,0.823679 1.17,0.88l3.3912e-08,-3.65162e-10c0.463892,-0.00499534 0.835902,-0.385104 0.830906,-0.848996c-0.000111337,-0.0103393 -0.000413572,-0.0206757 -0.000906549,-0.0310039" transform="translate(32, 68.6)"></path><path fill="none" stroke="#191816" d="M0,0l5.35,1.52" transform="translate(30.33, 64.59)"></path><path fill="#191816" d="M0.99,1.09647e-08h-0.43c-0.35,0 -0.56,0.33 -0.48,0.74l0.11,0.52l-1.82922e-09,-1.18423e-08c0.0605526,0.392016 0.375888,0.695066 0.77,0.74h0.44c0.34,0 0.55,-0.33 0.47,-0.74l-0.1,-0.51l-1.31727e-08,-7.75299e-08c-0.0669798,-0.39422 -0.383458,-0.698525 -0.78,-0.75Z" transform="translate(32.65, 65.35)"></path></g></svg>
```

Comment:
Should we pick a few more?

Reply:
@perry-mitchell yes! what kinds should I find?

Reply:
done